docs/_events/breadcrumbs.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Breadcrumbs
+events:
+  - name: link clicks
+    description: Triggered when a breadcrumb link is clicked, right clicked, shift clicked, control clicked, or windows key/command key clicked.
+    example_url: https://www.gov.uk/learn-to-drive-a-car
+    tracker: link_tracker
+    data:
+    - name: event
+      value: event_data
+    - name: event_data
+      value:
+      - name: event_name
+        value: navigation
+      - name: external
+        value: 'false'
+      - name: index
+        value:
+        - name: index_link
+        value: The position of the link within the breadcrumbs from left to right.
```

Comment:
This needs to be indented - `index` is using this as the value, and ignoring the sub attribute `index_link`.

![Screenshot 2023-03-27 at 09 40 19](https://user-images.githubusercontent.com/861310/227888968-e702ac0d-4cb5-4912-a545-a3d8ffb58fb0.png)